view/player/tmpl/default.html.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Nooku Framework - http://nooku.org/framework
+ *
+ * @copyright	Copyright (C) 2011 - 2014 Johan Janssens and Timble CVBA. (http://www.timble.net)
```

Comment:
Yes all the other files have the same header even in the master branch. What do I need to change it into?